.github/workflows/caching-build.yml

```diff
@@ -171,6 +171,27 @@ jobs:
       # no symlink following here (to avoid cycle`s)
       - run: sudo find '/home/runner/' -iname '*.ci.log' -exec mv {} ./log_save/ \;
         if: always()
+      # Download previous benchmark result from cache (if exists)
+      - name: Download previous benchmark data
```

Comment:
You're probably gonna want to use artifacts for this and not cache (we use cache alot, data's gonna get rotated out).